app/lib/associated_user_creator.rb

```diff
@@ -21,10 +21,10 @@
 class AssociatedUserCreator
   attr_reader :protocol_role
 
-  def initialize(params)
+  def initialize(params, current_identity)
```

Comment:
This new param will be missing from `app/controllers/associated_users_controller.rb`.

See `associated_users_controller.rb:70`
```creator = AssociatedUserCreator.new(project_role_params)```

Reply:
Fixed